ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/TransactionTransmitter.java

```diff
@@ -62,21 +63,25 @@ public TransactionTransmitter(
   }
 
   public void send() {
-    createSignedTransactionBody();
+    final Optional<JsonRpcRequest> request = createSignedTransactionBody();
+
+    if (request.isEmpty()) {
```

Comment:
what does it mean if it's empty? Should we log this?

Reply:
basically we pulled out transmission of the transaction from the func below - if it comes back empty, the issue has already been dealt with, and this just needs to exit.